training_config/multilang_dependency_parser.jsonnet

```diff
@@ -0,0 +1,142 @@
+                        "es": "https://www.dropbox.com/s/ypjuzlf7kj957g3/options262.json?dl=1",
+                        "fr": "https://www.dropbox.com/s/ypjuzlf7kj957g3/options262.json?dl=1",
+                        "it": "https://www.dropbox.com/s/ypjuzlf7kj957g3/options262.json?dl=1",
+                        "pt": "https://www.dropbox.com/s/ypjuzlf7kj957g3/options262.json?dl=1",
+                        "sv": "https://www.dropbox.com/s/ypjuzlf7kj957g3/options262.json?dl=1",
+                        "de": "https://www.dropbox.com/s/ypjuzlf7kj957g3/options262.json?dl=1"
+                    },
+                    "weight_files": {
+                        "en": "https://www.dropbox.com/s/1h62kc1qdcuyy2u/en_weights.hdf5?dl=1",
+                        "es": "https://www.dropbox.com/s/ygfjm7zmufl5gu2/es_weights.hdf5?dl=1",
+                        "fr": "https://www.dropbox.com/s/mm64goxb8wbawhj/fr_weights.hdf5?dl=1",
+                        "it": "https://www.dropbox.com/s/owfou7coi04dyxf/it_weights.hdf5?dl=1",
+                        "pt": "https://www.dropbox.com/s/ul82jsal1khfw5b/pt_weights.hdf5?dl=1",
+                        "sv": "https://www.dropbox.com/s/boptz21zrs4h3nw/sv_weights.hdf5?dl=1",
+                        "de": "https://www.dropbox.com/s/2kbjnvb12htgqk8/de_weights.hdf5?dl=1"
+                    }
+                }
+            }
+        }
+    },
+    "train_data_path": {
+        "de": "UNI_DEP_V2_PATH/universal/uni-dep-tb/universal_treebanks_v2.0/std/de/de-universal-train.conll",
+        "en": "UNI_DEP_V2_PATH/universal/uni-dep-tb/universal_treebanks_v2.0/std/en/en-univiersal-train.conll",
+        "fr": "UNI_DEP_V2_PATH/universal/uni-dep-tb/universal_treebanks_v2.0/std/fr/fr-universal-train.conll",
+        "it": "UNI_DEP_V2_PATH/universal/uni-dep-tb/universal_treebanks_v2.0/std/it/it-universal-train.conll",
+        "pt": "UNI_DEP_V2_PATH/universal/uni-dep-tb/universal_treebanks_v2.0/std/pt-br/pt-br-universal-train.conll",
+        "sv": "UNI_DEP_V2_PATH/universal/uni-dep-tb/universal_treebanks_v2.0/std/sv/sv-universal-train.conll"
+    },
+    "validation_data_path": {
+        "de": "UNI_DEP_V2_PATH/universal/uni-dep-tb/universal_treebanks_v2.0/std/de/de-universal-dev.conll",
+        "en": "UNI_DEP_V2_PATH/universal/uni-dep-tb/universal_treebanks_v2.0/std/en/en-univiersal-dev.conll",
+        "es": "UNI_DEP_V2_PATH/universal/uni-dep-tb/universal_treebanks_v2.0/std/es/es-universal-dev.conll",
+        "fr": "UNI_DEP_V2_PATH/universal/uni-dep-tb/universal_treebanks_v2.0/std/fr/fr-universal-dev.conll",
+        "it": "UNI_DEP_V2_PATH/universal/uni-dep-tb/universal_treebanks_v2.0/std/it/it-universal-dev.conll",
+        "pt": "UNI_DEP_V2_PATH/universal/uni-dep-tb/universal_treebanks_v2.0/std/pt-br/pt-br-universal-dev.conll",
+        "sv": "UNI_DEP_V2_PATH/universal/uni-dep-tb/universal_treebanks_v2.0/std/sv/sv-universal-dev.conll"
+    },
+    "test_data_path": {
```

Comment:
Oh, man, that's annoying - these are all in the same directory, so the strategy I recommended above doesn't work for the test data (for train and validation, you can just give the paths you want in the parameters for the dataset reader).  It'd be easier if these were laid out as `std/train/de/`, `std/dev/de/`, `/std/test/de/`, etc.